karton/core/resource.py

```diff
@@ -136,9 +136,6 @@ def sha256(self) -> str:
         :return: Hexencoded resource SHA256 hash
         """
         sha256 = self.metadata.get("sha256")
```

Comment:
```suggestion
        return self.metadata.get("sha256")
```

also this changes the return type